docs/Getting Started.md

```diff
@@ -747,8 +747,8 @@ Handling connection for 3003
 * Push data to the application. In order to do that follow the steps:
  * create a json file named `data.json` with the content `{"deviceId":"c75cb448-df0e-4692-8e06-0321b7703992","timestamp":1495545346279,"measurements":{"power":1.7,"rotorSpeed":23.4,"windSpeed":100.1}}`
  * run the following bash script:
- ```
- for str in $(cat test-data/10-storm.json)
+```
+ for str in $(cat data.json)
```

Comment:
There is no `data.json` under `examples/sensor-data-xxx\test-data` so this might be confusing when someone checks the actual code. Other than that LGTM.

Reply:
Thanks @skonto for the fast review.

This is funny ... I had assumed the `10-storm.json` file was a copy-paste relic from an old "Storm" project (as in "Big Data" by Nathan Marz). Turns out this is actually a "storm" in the domain of wind turbines ;-)

I have reverted this line to the original (I hope, not entirely sure, with editing in the Github UI).

I presume the conversation could be Resolved and this is good to merge then?

Reply:
@peter-vandenabeele-axa :) Yeah turbines face storms ;) It is ok to merge but I think you havent signed an SLA for the contribution because we didnt have that ready. I will delay a bit merging it, and will ping you to create the same PR again after we setup the SLA thing. Sorry for the inconvenience, but we will get it in.

Reply:
The CLA bot is working properly now. @peter-vandenabeele-axa could you open this PR again? We will approve right away.